ui/components/app/security-provider-banner-alert/blockaid-banner-alert/blockaid-banner-alert.js

```diff
@@ -88,41 +88,41 @@ function BlockaidBannerAlert({ txData, ...props }) {
     </Text>
   ) : null;
 
-  const isFailedResultType = resultType === BlockaidResultType.Failed;
-
   const severity =
     resultType === BlockaidResultType.Malicious
       ? Severity.Danger
       : Severity.Warning;
 
   const title = t(REASON_TO_TITLE_TKEY[reason] || 'blockaidTitleDeceptive');
 
-  const reportData = {
-    domain: origin ?? msgParams?.origin,
-    jsonRpcMethod: type,
-    jsonRpcParams: JSON.stringify(txParams ?? msgParams),
-    blockNumber: block,
-    chain: NETWORK_TO_NAME_MAP[chainId],
-    classification: reason,
-    blockaidVersion: BlockaidPackage.version,
-    resultType,
-    reproduce: JSON.stringify(features),
-  };
+  const reportUrl = (() => {
+    const reportData = {
+      blockNumber: block,
+      blockaidVersion: BlockaidPackage.version,
+      chain: NETWORK_TO_NAME_MAP[chainId],
+      classification: reason,
+      domain: origin ?? msgParams?.origin ?? txParams?.origin,
+      jsonRpcMethod: type,
+      jsonRpcParams: JSON.stringify(txParams ?? msgParams),
+      resultType,
+      reproduce: JSON.stringify(features),
+    };
```

Comment:
the only key/value prop updated here is domain. all others were reordered alphabetically